release/src/slack.ts

```diff
@@ -169,12 +169,25 @@ export function sendSlackMessage({ channelName = SLACK_CHANNEL_NAME, message }:
   });
 }
 
-async function getSlackChannelId(channelName: string) {
+async function getSlackChannelId(
+  channelName: string,
+  cursor?: string,
+): Promise<string | undefined> {
   const response = await slack.conversations.list({
+    cursor,
     limit: 9999,
     exclude_archived: true,
   });
-  return response.channels?.find((channel) => channel.name === channelName)?.id;
+
+  const maybeChannelId = response.channels?.find(
+    channel => channel.name === channelName,
+  )?.id;
+  const nextCursor = response.response_metadata?.next_cursor;
+  if (nextCursor) {
+    return maybeChannelId || (await getSlackChannelId(channelName, nextCursor));
```

Comment:
maybe just do something like:

```
if (!maybeChannelId && nextCursor) {
    return await getSlackChannelId(channelName, nextCursor);
}
```

?

Reply:
That's better! Thanks for the suggestion. Fixed in e93ff5853742fd04b1b868fe0b64e7e1c30a4f88